StringExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+
+namespace EntityManager
+{
+    public static class StringExtensions
+    {
+        public static string FirstCharToUpper(this string input)
+        {
+            if (!string.IsNullOrEmpty(input))
+            {
+                return input.First().ToString().ToUpper() + input.Substring(1);
+            }
+            return string.Empty;
+        }
```

Comment:
this function shouldnt work on empty string -> throw error/exception<br>
Maybe you should use ToCharArray(); method?<br>
var bla = input.ToCharArray();<br>
bla[0] = char.ToUpper(bla[0]);<br>
return bla.ToString(); // or return new string(bla); <br>
both are good -> however char method is much faster. matter of choice / performance

Reply:
Changed to char.ToUpper() and throwing error if string is null or empty